geoipupdate/test/test_geoip_update.py

```diff
@@ -51,8 +51,8 @@ def test_sha256_match(self, mock_urlopen='http://foo'):
 
     def test_shutil_raise(self):
         out_file = tempfile.NamedTemporaryFile(delete=False)
-        existing_file_as_string = 'random'
-        self.assertRaises(shutil.Error, write(out_file, existing_file_as_string))
+        existing_file_as_string = 'invalid/path/file'
```

Comment:
was this chnage needed after adding python 3.8?

Reply:
Yes, the test syntaxis was invalid and the shutil.Error was never raised since both files existed before.

Reply:
@mraarif Name updated.